assets/src/edit-story/constants.js

```diff
@@ -40,6 +40,11 @@ export const DEFAULT_EM = PAGE_HEIGHT * 0.02186;
 export const MIN_FONT_SIZE = 30;
 export const MAX_FONT_SIZE = 200;
 
+export const CTA_ZONE_PERCENT = 0.2; /** Bottom 20% */
```

Comment:
Do we have any reference to all four of these values somewhere in the Stories docs?